api/__tests__/index.test.js

```diff
@@ -0,0 +1,273 @@
+const { app, server } = require('../../server');
+const request = require('supertest');
+const promotionsJSON = require('../../data/promotions');
+
+afterAll(() => server.close());
+
+describe('Companies Endpoint', () => {
+  describe('/v1/companies/', () => {
+    it('Should respond to GET method', done => {
+      request(app)
+        .get('/v1/companies/')
+        .set('Accept', 'application/json')
+        .expect('Content-Type', /json/)
+        .expect(200, done);
+    });
+  });
+
+  describe('/companies/:companyId', () => {
+    it('Should respond to GET method', done => {
+      request(app)
+        .get('/v1/companies/1')
+        .set('Accept', 'application/json')
+        .expect('Content-Type', /json/)
+        .expect(res => {
+          expect(res.body).toEqual({ id: 1, name: 'Test Company' });
```

Comment:
Do assertions on individual key:value pairs instead of matching the whole JSON.
If this was intended to be a test for the API contract (schema validation) itself, I am sure there are better frameworks out there that do that automatically.